src/world/Spell/Customization/HackFixes.cpp

```diff
@@ -1127,6 +1127,16 @@ void SpellMgr::applyHackFixes()
                 // mask for FORM_CAT(1) = 1 << (1 - 1), which is 1
                 sp->setRequiredShapeShift(1);
             } break;
+            case 69075: // Bone Storm (Lord Marrowgar)
+            case 70834: // Bone Storm (Lord Marrowgar)
+            case 70835: // Bone Storm (Lord Marrowgar)
+            case 70836: // Bone Storm (Lord Marrowgar)
+            case 72864: // Death Plague (Rotting Frost Giant)
+            case 71160: // Plague Stench (Stinky)
+            case 71161: // Plague Stench (Stinky)
+            case 71123: // Decimate (Stinky & Precious)
+                sp->setEffectRadiusIndex(sSpellRadiusStore.LookupEntry(12)->ID, 0); // 100yd
```

Comment:
DBC data does not seem to have radius for these spells so this is needed, but 100 yards seem quite large radius